includes/push-ui.php

```diff
@@ -95,12 +95,12 @@ function ajax_push() {
 		/**
 		 * Add possibility to send notification in background
 		 *
-		 * @param bool  false   Whether run 'push' action in background or not, default 'false'
+		 * @param bool  true    Whether run 'push' action in background or not, default 'false'
 		 * @param array $params request data
 		 */
-		$push_in_background = apply_filters( 'dt_push_allow_in_background', false, $params );
+		$allow_push = apply_filters( 'dt_allow_push', true, $params );
```

Comment:
add a sentence describing how this filter could be used

Reply:
In case the 'push' action is consuming a lot of resources, we can use the `dt_allow_push` filter to terminate function execution, and run the function in background. Here is simple use case example:

    add_filter( 'dt_allow_push', 'schedule_push_action', 10, 2 );
    
    function schedule_push_action( $allow_push, $params ) {
	if ( ! wp_next_scheduled( 'dt_push_posts_hook' ) ) {
		wp_schedule_single_event( time(), 'dt_push_posts_hook', [ $params ] );
	}

	return false;
    }

And then simply we can run it by hooking to scheduled event:

    add_action( 'dt_push_posts_hook', 'push_action', 10, 1 );

    function push_action( $params ) {
	\Distributor\PushUI\push( $params );
    }

Or we can call custom function depending on use case.